akka-operator/values.yaml

```diff
@@ -38,3 +42,21 @@ resources:
 nodeSelector: {}
 tolerations: []
 affinity: {}
+# valid values:
```

Comment:
we can remove this section as well, I would make a flag like: `ubbAgentEnabled` WDYT?

Reply:
@andreaTP So you are saying that we replace all checks on provider with checking a flag like `ubbagentEnabled` ?

Reply:
@andreaTP I have kept the `provider` and all gcp related stuff are based on the conditional `provider == gcp`. Think `ubbagentEnabled` is a bit vague and may be confusing too if tomorrow we introduce some other conditionals